### PR TITLE
[NG] Single Member Profile page: Include project badges

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Description**

Enhancement to the single page view.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Jashpatel1 — DISMISSED** [review]
Looks good to me. Just worried about the package-lock.json changes

**Comment by leecalcote** [comment]
> Looks good to me. Just worried about the package-lock.json changes

Good point. Trying to move past not being able to build the project, I deleted my cloned copy of the repo and started fresh. `package-lock.json` rebuilt in the process.

File: src/collections/members/olushesi-toheeb/index.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: person
+name: Olushesi Toheeb
```
Comment (OlushesiToheeb) [inline src/collections/members/olushesi-toheeb/index.mdx:3]: has this been on layer5-ng too?

**Review by OlushesiToheeb — COMMENTED** [review]
It's all good at @leecalcote. Thanks.

File: src/collections/members/olushesi-toheeb/index.mdx (hunk 1 above)
Comment (leecalcote) [reply src/collections/members/olushesi-toheeb/index.mdx:3]: @OlushesiToheeb just now being created in this PR, yes.

**Pull request merged**